nominees/opensrp.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "OpenSRP",
+  "description": "OpenSRP is an open-source electronic health record platform designed for use by frontline health workers in low connectivity environments on basic Android devices. It supports data collection, client management, reporting, and can integrate with many health information system platforms.",
+  "website": "https://company.ona.io/products/opensrp-enterprise/features/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/onaio/rdt-standard/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceURL": "https://smartregister.atlassian.net/wiki/spaces/Documentation/overview"
+    },
+    {
+      "SDGNumber": 17,
+      "evidenceURL": "https://smartregister.atlassian.net/wiki/spaces/Documentation/overview"
```

Comment:
@rachita19082 here is a more relevant [link](https://socialdigital.iadb.org/en/sph/covid-19/digital-solutions/4770) to support SDG 17. Kindly update or feel free to share one which provides stronger evidence.

Reply:
Updated in commit e5ff53b